.dev/tests/phpunit/includes/test-coblocks-block-assets.php

```diff
@@ -98,7 +98,31 @@ public function test_editor_assets_scripts() {
 		$this->go_to( '/wp-admin/post-new.php' );
 		$this->coblocks_block_assets->editor_assets();
 
-		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered ) );
+		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered,  ) );
+
+		$path = 'wp-content/plugins/coblocks/dist/coblocks-*.js';
+		$files = glob( $path );
+
+		foreach($files as $file) {
+			$dist_script = end(
+				explode('/',
+					explode('.',
+						$file
+					)[0]
+				)
+			);
+
+			if ($dist_script == 'coblocks-plugin-deactivation') {
```

Comment:
Spacing, just to keep consistent with the other code.

```suggestion
			if ( $dist_script == 'coblocks-plugin-deactivation' ) {
```